### PR TITLE
add option to use arrival time in nearby view

### PR DESCRIPTION
**Pull request by daniel-heppner-ibigroup**
**Description:**
Adds configuration option to use the arrival time rather than departure time in Nearby View and other places where the patternrow is used.

### CODE REVIEW

File: lib/util/viewer.js [hunk 1]
```diff
@@ -13,12 +13,20 @@ import { isBlank } from './ui'
...
+  useSchedule,
+  useArrivalTime
+) {
+  const time = useSchedule
+    ? useArrivalTime
+      ? stopTime.scheduledArrival
+      : stopTime.scheduledDeparture
+    : useArrivalTime
+    ? stopTime.realtimeArrival
     : stopTime.realtimeDeparture
```
Comment (miles-grant-ibigroup) [inline lib/util/viewer.js]: Oy vey

**Review by miles-grant-ibigroup — APPROVED** [review]
Will leave it up to the second reviewer to decide if that ternary is alright

File: lib/util/viewer.js (hunk 1 above)
Comment (amy-corson-ibigroup) [reply lib/util/viewer.js]: I guess if it works I don't mind the long ternary, but if we could add a comment explaining the logic, just so it's not a headache to parse in the future, that'd be great!!

**Review by amy-corson-ibigroup — APPROVED** [review]
Beautiful new config item sorry for long delay in reviewing

**Pull request merged**